apps/web/components/v2/ui/NavTabs.tsx

```diff
@@ -0,0 +1,96 @@
+import { AdminRequired } from "components/ui/AdminRequired";
+import noop from "lodash/noop";
```

Comment:
why do we need lodash

Reply:
Instead of `noop` just do `() => {}` @agustif - never ever found a good usecase for noop.

Reply:
Will reuse the ones already done!